### PR TITLE
Create Insert-sort.py

### PR DESCRIPTION
**Pull request by ugin331**

### CODE REVIEW

File: Insert-sort.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+for i in range(10):
+    list1.append(random.randint(1, 10))
+#print the starting list.
+for i in reversed(range(len(list1))):
+    print(i, ": ", list1[i])
+for i in range(len(list1)):
+    #set a temporary variable to go through each number in the list, starting on the one in pos i, then going down.
+    tmp_var = i
+    #do a while loop to run through each previous number, and it stops when either tmp_var becomes 0, or if the current number becomes less than the one before it
+    while tmp_var > 0 and list1[tmp_var]> list1[tmp_var-1]:
```
Comment (davidxl) [inline Insert-sort.py]: add space before >

File: Insert-sort.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        #swap numbers
+        list1[tmp_var], list1[tmp_var-1] = list1[tmp_var-1], list1[tmp_var]
+        #move one down the list
+        tmp_var -= 1
+        print(list1)
+print(list1[::-1])
```
Comment (davidxl) [inline Insert-sort.py]: why print it in reverse order?

Comment (ugin331) [reply Insert-sort.py]: it makes the list in reverse order, so it makes it backwards to print it properly sorted

File: Insert-sort.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+list1 = []\
+#append 10 random numbers into the list.
+for i in range(10):
+    list1.append(random.randint(1, 10))
+#print the starting list.
+for i in reversed(range(len(list1))):
+    print(i, ": ", list1[i])
+for i in range(len(list1)):
+    #set a temporary variable to go through each number in the list, starting on the one in pos i, then going down.
+    tmp_var = i
```
[the remark this answers is not in the document]
Comment (ugin331) [reply Insert-sort.py]: ok

File: Insert-sort.py (hunk 1 above)
Comment (ugin331) [reply Insert-sort.py]: This is a formatting thing, right?

**Comment by ugin331** [comment]
So can I merge?

File: Insert-sort.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidxl) [reply Insert-sort.py]: did you update the branch ?

**Pull request merged**